packages/brand-moment/src/BrandMoment.spec.tsx

```diff
@@ -0,0 +1,59 @@
+import { Box, Paragraph } from "@kaizen/component-library"
+import { cleanup, render } from "@testing-library/react"
+import * as React from "react"
+import { MinimalNavigation } from "@cultureamp/unified-navigation-ui"
+import { BrandMomentError } from "@kaizen/draft-illustration"
+import { BrandMoment } from "./BrandMoment"
+
+window.matchMedia = jest.fn().mockImplementation(() => ({
+  matches: false,
+  media: "",
+  onchange: null,
+  addListener: jest.fn(),
+  addEventListener: jest.fn(),
+  removeListener: jest.fn(),
+  removeEventListener: jest.fn(),
+}))
+
+// Stub video elements for JS-Dom to prevent console errors
+window.HTMLMediaElement.prototype.load = () => jest.fn()
+// @ts-ignore-next-line
```

Comment:
I know this looks bad, but for a stub I thought it'd be okay? Without it the type is a bit strange:
```
Type '() => jest.Mock<any, any>' is not assignable to type '() => Promise<void>'.
  Type 'Mock<any, any>' is missing the following properties from type 'Promise<void>': then, catch, finally, [Symbol.toStringTag]
```

Reply:
Typescript is complaining because `play` returns a Promise, not a function. In this scenario, we can trust that the illustration package is properly testing that `play` is call on mount, so you don't need to mock a reject/resolve. Something like this would work: 
`window.HTMLMediaElement.prototype.play = () => new Promise(jest.fn())`